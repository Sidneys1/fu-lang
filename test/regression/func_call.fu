test: u8() => 42;

main: u8() => test();
