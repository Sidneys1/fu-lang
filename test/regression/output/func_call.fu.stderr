INFO:fu.compiler:Output binary `a.out`: 352 Bytes
